distributionviewer/core/static/js/app/components/views/example-date-range.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function(props) {
+  return (
+    <span className="date-range">
+      <time dateTime={props.from}>{props.from}</time> to <time dateTime={props.to}>{props.to}</time>
```

Comment:
For some reason, all attributes need to be camel-cased in React. Lowercase doesn't have any effect.